tests/integration_test.js

```diff
@@ -18,11 +18,13 @@ fse.mkdirSync(tmpDir);
 process.chdir(tmpDir);
 
 const CF = new AWS.CloudFormation({ region: 'us-east-1' });
+const templateName = 'aws-nodejs';
 BbPromise.promisifyAll(CF, { suffix: 'Promised' });
 
 describe('Service Lifecyle Integration Test', () => {
   it('should create service in tmp directory', () => {
-    execSync(`${serverlessExec} create --template aws-nodejs`, { stdio: 'inherit' });
+    execSync(`${serverlessExec} create --template ${templateName}`, { stdio: 'inherit' });
+    execSync(`sed -i s/${templateName}/$RANDOM/g serverless.yaml`);
```

Comment:
Just ran it locally. This command succeeded: `sed -i '' "s/aws-nodejs/$RANDOM/g" serverless.yaml`